qt/aqt/editor.py

```diff
@@ -1553,6 +1553,15 @@ def dropEvent(self, evt: QDropEvent | None) -> None:
         extended = self._wantsExtendedPaste()
         mime = evt.mimeData()
         assert mime is not None
+
+        if (
+            self.editor.state is EditorState.IO_PICKER
+            and (html := self._processUrls(mime, allowed_suffixes=pics))
+            and (path := self.editor.extract_img_path_from_html(html))
+        ):
+            self.editor.setup_mask_editor_for_new_note(path)
```

Comment:
To make it work when "Reset Image Occlusion" is used:

```suggestion
            self.editor.setup_mask_editor(path)
```